docs/notebooks/using_mclip.ipynb

```diff
@@ -0,0 +1,328 @@
+    "    epochs=5,\n",
+    "    learning_rate=1e-6,\n",
+    "    loss='CLIPLoss',\n",
+    "    device='cuda',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6be36da7-452b-4450-a5d5-6cae84522bb5",
+   "metadata": {},
+   "source": [
+    "You may notice that this piece of code looks very similar to the one used to fine-tune regular clip models, as shown [here](https://finetuner.jina.ai/notebooks/text_to_image/). The only real difference is the data being provided and the model being used. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "923e4206-ac60-4a75-bb3d-4acfc4218cea",
+   "metadata": {},
+   "source": [
+    "## Monitoring\n",
+    "\n",
+    "Now that we've created a run, let's see its status. You can monitor the run by checking the status - `run.status()` and - the logs - `run.logs()` or - `run.stream_logs()`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56d020bf-8095-4a83-a532-9b6c296e985a",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# note, the fine-tuning might takes 20~ minutes\n",
+    "for entry in run.stream_logs():\n",
+    "    print(entry)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b58930f1-d9f5-43d3-b852-5cbaa04cb1aa",
+   "metadata": {},
+   "source": [
+    "Since some runs might take up to several hours/days, it's important to know how to reconnect to Finetuner and retrieve your run.\n",
+    "\n",
+    "```python\n",
+    "import finetuner\n",
+    "\n",
+    "finetuner.login()\n",
+    "run = finetuner.get_run(run.name)\n",
+    "```\n",
+    "\n",
+    "You can continue monitoring the run by checking the status - `finetuner.run.Run.status()` or the logs - `finetuner.run.Run.logs()`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0b81ec1-2e02-472f-b2f4-27085bb041cc",
+   "metadata": {},
+   "source": [
+    "## Evaluating\n",
+    "Currently, we don't have a user-friendly way to get evaluation metrics from the {class}`~finetuner.callback.EvaluationCallback` we initialized previously.\n",
+    "\n",
+    "```bash\n",
+    "           INFO     Done ✨                                                                              __main__.py:219\n",
+    "           INFO     Saving fine-tuned models ...                                                         __main__.py:222\n",
+    "           INFO     Saving model 'model' in /usr/src/app/tuned-models/model ...                          __main__.py:233\n",
+    "           INFO     Pushing saved model to Jina AI Cloud ...                                                    __main__.py:240\n",
+    "[10:38:14] INFO     Pushed model artifact ID: '62a1af491597c219f6a330fe'                                 __main__.py:246\n",
+    "           INFO     Finished 🚀                                                                          __main__.py:248\n",
+    "```\n",
+    "\n",
+    "```{admonition} Evaluation of CLIP\n",
```

Comment:
This doesn't seem to be rendering correctly.

Reply:
same as above